in_toto/keylib_test.go

```diff
@@ -16,17 +16,17 @@ func TestLoadKey(t *testing.T) {
 		hashAlgorithms []string
 		expectedKeyID  string
 	}{
-		{"rsa public key", "alice.pub", "rsassa-pss-sha256", []string{"sha256", "sha512"}, "556caebdc0877eed53d419b60eddb1e57fa773e4e31d70698b588f3e9cc48b35"},
-		{"rsa private key", "dan", "rsassa-pss-sha256", []string{"sha256", "sha512"}, "b7d643dec0a051096ee5d87221b5d91a33daa658699d30903e1cefb90c418401"},
-		{"rsa public key", "dan.pub", "rsassa-pss-sha256", []string{"sha256", "sha512"}, "b7d643dec0a051096ee5d87221b5d91a33daa658699d30903e1cefb90c418401"},
-		{"ed25519 private key", "carol", "ed25519", []string{"sha256", "sha512"}, "be6371bc627318218191ce0780fd3183cce6c36da02938a477d2e4dfae1804a6"},
-		{"ed25519 public key", "carol.pub", "ed25519", []string{"sha256", "sha512"}, "be6371bc627318218191ce0780fd3183cce6c36da02938a477d2e4dfae1804a6"},
-		{"ecdsa private key (P521)", "frank", "ecdsa-sha2-nistp521", []string{"sha256", "sha512"}, "434cf7c5b168f6ea4c7e6e67afa74a02625310530f1664f761637bdc7ad8f8df"},
-		{"ecdsa public key (P521)", "frank.pub", "ecdsa-sha2-nistp521", []string{"sha256", "sha512"}, "434cf7c5b168f6ea4c7e6e67afa74a02625310530f1664f761637bdc7ad8f8df"},
-		{"ecdsa private key (P384)", "grace", "ecdsa-sha2-nistp384", []string{"sha256", "sha512"}, "a5522ebccd492f64e6ec0bbcb5eb782708f6e26709a3712e64fff108b98e5142"},
-		{"ecdsa public key (P384)", "grace.pub", "ecdsa-sha2-nistp384", []string{"sha256", "sha512"}, "a5522ebccd492f64e6ec0bbcb5eb782708f6e26709a3712e64fff108b98e5142"},
-		{"ecdsa private key (P224)", "heidi", "ecdsa-sha2-nistp224", []string{"sha256", "sha512"}, "fae849ef9247cc7d19ebd33ab63b5d18a31357508fd82d8ad2aad6fdcc584bd7"},
-		{"ecdsa public key (P224)", "heidi.pub", "ecdsa-sha2-nistp224", []string{"sha256", "sha512"}, "fae849ef9247cc7d19ebd33ab63b5d18a31357508fd82d8ad2aad6fdcc584bd7"},
+		{"rsa public key", "alice.pub", "rsassa-pss-sha256", []string{"sha256", "sha512"}, "d7b728368798278c6bbd43e57b9ff9794be73c24edc574fdaae67efcbc34e23a"},
```

Comment:
Was generating new pub keys necessary for the PR?

Reply:
There's a few things going on here, first is that due to adding the `certificate` field to the KeyVal struct all key ids changed.  I had previously attempted to overload the public field and use that for the certificate but it proved difficult to manage.

Alice's key did have to be regenerated due to her private not being checked into the repo.  I needed to re-sign the test data so I needed Alice's private key.

We can revisit the KeyVal struct changes and attempt to find a way to facilitate these changes without changing key ids if wanted.  It's not ideal to have to update all key ids obviously.

Reply:
> Alice's key did have to be regenerated due to her private not being checked into the repo. I needed to re-sign the test data so I needed Alice's private key.

Oh, this must have been my fault sorry. Looks like I forgot adding it to the repository when I was working on the in-toto run implementation.

I know what you mean, but I think getting this merged is priority number one right now + I would prefer better code coverage over the key ID issue.

Reply:
I will definitely focus more unit tests today to get that coverage up